src/style.css

```diff
@@ -0,0 +1,115 @@
+html {
```

Comment:
You don't need this file anymore, delete it

Reply:
but in this file i do reset and import